src/components/MovieList/MovieList.jsx

```diff
@@ -1,5 +1,8 @@
+import { MovieCard } from '../MovieCard';
 import './MovieList.scss';
 
-export const MovieList = () => (
-  <>Put the list here</>
+export const MovieList = ({ movies }) => (
+  <div className="movies">
```

Comment:
let's use semantic tags for list
P.S. make sure you fixed styles after changing tags